src/main/java/com/exadel/discount/controller/VendorController.java

```diff
@@ -1,90 +1,76 @@
 package com.exadel.discount.controller;
 
-import com.exadel.discount.dto.location.CreateLocationDTO;
-import com.exadel.discount.dto.location.LocationInfoDTO;
-import com.exadel.discount.dto.vendor.BaseVendorDTO;
-import com.exadel.discount.dto.vendor.CreateVendorDTO;
-import com.exadel.discount.dto.vendor.DetailedVendorDTO;
+import com.exadel.discount.dto.VendorDTO;
+import com.exadel.discount.dto.VendorLocationDTO;
+import com.exadel.discount.dto.validation.Create;
+import com.exadel.discount.service.interfaces.VendorLocationService;
+import com.exadel.discount.service.interfaces.VendorService;
 import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
 import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.*;
 
-import javax.validation.Valid;
-import javax.validation.constraints.Min;
-import javax.validation.constraints.Size;
-import java.util.ArrayList;
+import javax.validation.constraints.NotNull;
 import java.util.List;
+import java.util.UUID;
 
 @Validated
 @RestController
+@RequiredArgsConstructor
 @RequestMapping("/vendors")
 public class VendorController {
-    private final List<DetailedVendorDTO> detailedVendorsServiceMock = List.of(
```

Comment:
It's Nikolay's part, maybe it got in PR by mistake. It won't work because of absence of other vendor classes